Tests/HttpServerSpec/ActionsSpec/NullActionSpec.swift

```diff
@@ -33,7 +34,8 @@ class NullActionSpec: QuickSpec {
                                   "Content-Type": "text/html",
                                   "Allow": "",
                                   "Location": "",
-                                  "WWW-Authenticate": ""],
+                                  "WWW-Authenticate": "",
+                                  "Set-Cookie": ""],
```

Comment:
Does the response need to send back these blank headers? Is that one of the things that cobspec is checking? If not, you can just not send back a header if it is blank.

Reply:
This is not for cobspec. It's just the way I set up my response builder. It automatically generate this value if I don't specify any value.